addons/friendly_tracker/functions/fnc_loop.sqf

```diff
@@ -31,6 +31,9 @@ if (!([player] call FUNC(canTrack))) exitWith {
     [FUNC(loop), [], GVAR(refreshRate)] call CBA_fnc_waitAndExecute;
 };
 
+// Determine which sides we want to track
+GVAR(trackedSides) = call FUNC(getTrackedSides);
```

Comment:
As an alternative to doing it in the loop it could be done when related CBA settings are changed (`showSideCivilian` and `showSides`) and in CBA Player Event - Unit

Reply:
I thought about this but then we don't cover situation when side relations change or player's side change. It would need to be handled separately somehow.

Reply:
Leave it in PFH then.